src/locales/en-us/components/index.yaml

```diff
@@ -1,4 +1,4 @@
 components:
   route-view:
     title: "{name} Manager"
-    route-announcer: Navigated to {title}
+    route-announcer: Navigated to %s
```

Comment:
**Question**: What’s this change for? I kind of really don’t want to introduce a second interpolation syntax.

Reply:
I can't for the life of me remember why I did this, maybe it was left over from a previous experiment or something. I change it to use default `t` interpolation, if something comes up that reminds me why it was like this I can bring it up again.